modules/gtag-test/bus/listeners/listenProductDetail.ts

```diff
@@ -0,0 +1,5 @@
+import ProductDetail from '#ioc/bus/events/ProductDetail'
+
+export default (product: ProductDetail) => {
```

Comment:
Because event emitters are composables, make event listeners composables to so we can use stores, router and other functionalities in them.